pkg/capabilities/capabilities.go

```diff
@@ -80,6 +80,7 @@ type TriggerEvent struct {
 	TriggerType string
 	ID          string
 	Timestamp   string
+	Metadata    values.Value
```

Comment:
I was thinking that Metadata is reserved for Workflow-specific data, not trigger-specific (see comment below). Having said this, if the capability response type is report, then it might make sense to include the signers as part of the metadata. This should be documented, though.

Reply:
Also, shouldn't this be on the CapabilityResponse and not TriggerEvent?

Reply:
I want it to be trigger-specific, I will move the comment to apply to both fields.
I initially wanted to put it inside payload but didn't want to add a yet another layer of wrapping.